crates/sui-protocol-config/src/lib.rs

```diff
@@ -3254,7 +3283,19 @@ impl ProtocolConfig {
                         cfg.feature_flags.passkey_auth = true;
                     }
                 }
-                76 => {}
+                76 => {
+                    cfg.feature_flags.move_native_context = false;
+
+                    cfg.tx_context_fresh_id_cost_base = Some(30);
```

Comment:
This should be the same as `tx_context_derive_id_cost_base`. You could even reuse that one instead of this one possibly if you wanted to?

Reply:
I had it that way originally but the API is not identical and figured we would have one for each....